virt_lightning/shell.py

```diff
@@ -273,9 +272,9 @@ def main():
     except FileNotFoundError:
         print("Configuration file not found")
         exit(1)
-    except:
+    except Exception as e:
```

Comment:
flake might fail it because variable has so short name.

Reply:
It passes just fine.